variable/technology/technologies.yaml

```diff
@@ -460,3 +531,326 @@ Investment|Energy Supply|Liquids|Oil:
+                plant category.
+   unit: €/MWh
+
+Output Energy of the Second Block|Electricity|Coal:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Energy System Storage:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Gas:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Hydro:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Nuclear:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Oil:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Wind:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Output Energy of the Second Block|Electricity|Solar:
+   description: Generator output of the second block (i.e., above the minimum load)
+   unit: GW
+
+Ramp Up|Electricity:
+   description: Upward ramp limit of the power generation unit
+   unit: GW/h
+
+Ramp Up|Electricity|Coal:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Gas:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Nuclear:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Up|Electricity|Oil:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity:
+   description: Downward ramp limit of the power generation unit
+   unit: GW/h
+
+Ramp Down|Electricity|Coal:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Gas:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Nuclear:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Ramp Down|Electricity|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Reactance|Electricity|Transmission|Line:
```

Comment:
Is there reactance for anything else than electricity transmission lines?

Reply:
Yeah. There is the possibility to include power transformers, HVDC converter stations, etc.
I specified the format of this variable in the case of another energy model would have other types of equipment.

Reply:
ok, thanks for the explanation!